fastlane/lanes/screenshots.rb

```diff
@@ -61,10 +70,10 @@
         concurrent_simulators: true,
 
         devices: [
-          'iPhone Xs Max',
-          'iPhone 8 Plus',
-          'iPad Pro (12.9-inch) (2nd generation)',
-          'iPad Pro (12.9-inch) (3rd generation)'
+          'iPhone 13 Pro Max',
+          'iPhone 11 Pro',
+          'iPad Pro (12.9-inch) (4th generation)',
```

Comment:
[CI couldn't find this Simulator](https://buildkite.com/automattic/wordpress-ios/builds/8180#01813776-56f5-4f18-8894-0fc249570125/890-8212):

![image](https://user-images.githubusercontent.com/1218433/172103370-d60ab22a-3de8-41e4-aa88-fd0ce19c651a.png)

I'm not sure whether we should create it or change simulator type.

Reply:
Yeah I thought about creating it as well (`xcrun simctl create` and so on, actually Fastlane has a FourFlusher gem — and probably some actions as well — for that so there's probably something that can be done as well there… I'm surprised that action taking the screenshots doesn't create the sims by default already btw…).

Will definitively need to explore that.